src/main/java/org/folio/rest/impl/SamlAPI.java

```diff
@@ -751,7 +814,7 @@ private boolean isInvalidOrigin(String origin) {
 
   static String getCqlUserQuery(String userPropertyName, String value) {
     // very sad that RMB does not have an option to reject fields with no index
-    List<String> supported = List.of("barcode", "externalSystemId", "id", "username", "personal.email");
+    List<String> supported = List.of(BARCODE, EXTERNAL_SYSTEM_ID, ID, USERNAME, PERSONAL_EMAIL);
```

Comment:
maybe lets intorduce new enum with name UserSeachField, and here we can use stream filter operation

Reply:
I moved these constants to a new util class 'UserFields'